MapboxNavigation-Documentation.podspec

```diff
@@ -32,7 +32,7 @@ Pod::Spec.new do |s|
 
   # ――― Source Code ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
 
-  s.source_files = ["MapboxNavigation/*", "MapboxCoreNavigation/*"]
+  s.source_files = ["MapboxNavigation/**/*.{h,m,swift}","MapboxCoreNavigation"]
```

Comment:
Not sure what happened, but previous version did not find any source files. This line is composed from corresponding `MapboxNavigation` and `MapboxCoreNavigation` podspecs

Reply:
Interesting, I’m not sure why the previous syntax stopped working. The source files are still directly within these folders, and I’m unaware of any source file ever being in a subfolder of MapboxNavigation/.

As of #2230, there should no longer be any public C or Objective-C headers in the project. Can you remove the .m and .h bits from this file and MapboxNavigation.podspec? You’ll need to make one symbol private to silence a warning: https://github.com/mapbox/mapbox-navigation-ios/issues/2314#issuecomment-596593196.